marklogic-data-hub/src/testFixtures/java/com/marklogic/hub/test/AbstractHubTest.java

```diff
@@ -529,6 +529,26 @@ protected RunFlowResponse runFlow(FlowInputs flowInputs) {
         return response;
     }
 
+    protected void waitForReindex(HubConfig hubConfig, String database){
+        String query = "(\n" +
+            "  for $forest-id in xdmp:database-forests(xdmp:database('" + database + "'))\n" +
+            "  return xdmp:forest-status($forest-id)//*:reindexing\n" +
+            ") = fn:true()";
+        String status;
+        int attempts = 0;
+        do{
+            sleep(15000L);
```

Comment:
The test keeps failing in the pipeline even though it passed in my local. I am increasing the sleep time to see of if the test passes. I will then work to see why the query isn't reliable.

Reply:
You could try running this as a little main program (not the test itself, just this code - we don't want to clear the database) against the HC nightly index, just to test against an ML instance that isn't running on your laptop. I recommend checking with the DB team too to see if there's a recommended script for checking to see if the database is reindexing or not.